### PR TITLE
Utilize line-height in rebranded pages

### PR DESCRIPTION
**Pull request by CedrikNikita**
part of #1049 (or even fixes the whole issue what do you think @etharner?)

### CODE REVIEW

File: src/popup/router/components/AccountInfo.vue [hunk 1]
```diff
@@ -255,11 +255,8 @@ export default {
     justify-content: center;
     margin-top: 8px;
     margin-bottom: 6px;
-    line-height: 21px;
 
-    @extend %face-sans-14-medium;
```
[the remark this answers is not in the document]
Comment (etharner) [reply src/popup/router/components/AccountInfo.vue:260]: Yes, that's what Paolo requested until designs will cover long addresses issues

File: src/popup/router/components/AuctionCard.vue [hunk 2]
```diff
@@ -49,8 +50,8 @@ export default {
   }
 
   .name {
-    font-weight: bold;
```
Comment (etharner) [inline src/popup/router/components/AuctionCard.vue:52]: Looks like in designs we still need bold here 
image

**Review by etharner — COMMENTED** [review]
In general I support this changes, but there are still many places where we have something like that:
image
And this maybe should be resolved on design side first: it needs to be standardized and consistent to prevent further misusing of fonts.

File: src/popup/router/components/AuctionCard.vue (hunk 2 above)
Comment (CedrikNikita) [reply src/popup/router/components/AuctionCard.vue:52]: Was going to use %face-sans-14-bold here.

**Review by etharner — APPROVED** [review]

**Pull request merged**